config/workflows/wasCrawlDisseminationWF.xml

```diff
@@ -7,18 +7,6 @@
     <prereq>start</prereq>
     <label>Extracts WARC files from WACZ files</label>
   </process>
-  <process name="cdx-generator">
-    <prereq>warc-extractor</prereq>
-    <label>Generate CDX files for (W)ARCs</label>
-  </process>
-  <process name="cdx-merge-sort-publish">
-    <prereq>cdx-generator</prereq>
-    <label>Sort individual new CDX files, merge into existing openwayback cdx file</label>
-  </process>
-  <process name="path-indexer">
-    <prereq>cdx-merge-sort-publish</prereq>
-    <label>Update the path index with the new (W)ARCs</label>
-  </process>
   <process name="cdxj-generator">
     <prereq>path-indexer</prereq>
```

Comment:
Will need warc-extractor as the previous step since path-indexer no longer exists

Reply:
@lwrubel Ah, yes, thank you 😄